rapids_build_backend/utils.py

```diff
@@ -1,13 +1,10 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 
 import os
-from functools import lru_cache
 
 import tomli
 
 
-# Avoid unnecessary I/O by caching.
-@lru_cache
```

Comment:
I guess this is no longer valid because we're calling this multiple times after changing the contents in the middle via dfg?

Reply:
That is correct.